server/lib/src/server/migrations.rs

```diff
@@ -458,6 +458,7 @@ impl<'a> QueryServerWriteTransaction<'a> {
             SCHEMA_ATTR_CREDENTIAL_UPDATE_INTENT_TOKEN.clone().into(),
             SCHEMA_ATTR_DEVICEKEYS.clone().into(),
             SCHEMA_ATTR_DISPLAYNAME.clone().into(),
+            SCHEMA_ATTR_EC_KEY_PRIVATE.clone().into(),
```

Comment:
I alphabetised these! (ok, except for mail but it was being weird.) Please put it in the right place 😃

Reply:
wopsies